gradle-metric-schema/src/main/java/com/palantir/metric/schema/gradle/CreateMetricsManifestTask.java

```diff
@@ -43,12 +43,14 @@
 import org.gradle.api.logging.Logging;
 import org.gradle.api.provider.Property;
 import org.gradle.api.provider.Provider;
+import org.gradle.api.tasks.CacheableTask;
 import org.gradle.api.tasks.Classpath;
 import org.gradle.api.tasks.InputFile;
 import org.gradle.api.tasks.Optional;
 import org.gradle.api.tasks.OutputFile;
 import org.gradle.api.tasks.TaskAction;
 
+@CacheableTask
 public class CreateMetricsManifestTask extends DefaultTask {
```

Comment:
Do we need to do anything with @InputFile sensitivity? (I'm pattern matching on the conjure gradle plugin, I don't know much about how these things work!)